src/routes/safe/store/actions/processTransaction.js

```diff
@@ -24,23 +23,37 @@ import {
 } from '~/logic/notifications'
 import { getErrorMessage } from '~/test/utils/ethereumErrors'
 
+// TODO: remove
```

Comment:
what's this about?

Reply:
hmm I think I forgot to finish what I wanted to write ... should remove it, since I cannot remember